app/assets/javascripts/fae/form/_ajax.js

```diff
@@ -121,7 +121,33 @@ Fae.form.ajax = {
 
         // remotipart returns data inside textarea, let's grab it from there
         if ($html.is('textarea')) {
-          $html = $( $html.val() );
+          if ( $html.length > 1 ) {
+            var textarea;
+
+            var isTextarea = function(items) {
```

Comment:
1. A function like this should be defined outside of this `if` block, like above the `ajax:success`. 
2. `textarea` can be assigned on creation.  (L125 and L141)
3. Executing HTML of a textarea seems unsafe (L146). Is `$( textarea.text() )` insufficient? 
4. If this original problem stems from `.is('textarea')` being an Array, should L123 check for this to be `$.isArray()`? And that should remove the `if` on L124 and the need for the extra nesting on L145-50.